packages/fast-components/src/flipper/flipper.stories.ts

```diff
@@ -0,0 +1,13 @@
+import { FASTFlipper } from "./";
```

Comment:
As a naming standard, we already know in these stories, classes, etc, that we're dealing with FAST.  Do we need to replicate FAST in the named imports, interfaces, and class names?  Does this help the consumer?  Could we eliminate the prefixed naming for FAST?  We may be able to reduce file size while increasing maintainability in that it's easy to scan the file and understand what's going on.  Just from my fresh perspective.

Reply:
This is importing what is exported to the consumer which I think we do want name spaced. If we want to revisit naming of what is being exported I'd like to raise that as an issue in the repo so we can track independently of this PR.